src/WarcraftLegacies.Source/Quests/BlackEmpire/QuestBladeoftheBlackEmpire.cs

```diff
@@ -0,0 +1,40 @@
+﻿using MacroTools.ArtifactSystem;
+using MacroTools.FactionSystem;
+using MacroTools.LegendSystem;
+using MacroTools.ObjectiveSystem.Objectives.ControlPointBased;
+using MacroTools.ObjectiveSystem.Objectives.LegendBased;
+using MacroTools.QuestSystem;
+
+namespace WarcraftLegacies.Source.Quests.BlackEmpire
+{
+
+  public sealed class QuestBladeoftheBlackEmpire : QuestData
+  {
+
+    /// <summary>
+    /// Initializes a new instance of the <see cref="QuestBladeoftheBlackEmpire"/> class.
+    /// </summary>
+    public QuestBladeoftheBlackEmpire() : base("The Blade of the Black Empire",
+      "An ancient blade from the time of the Black Empire itself. The Dagger has a mind of its own and tries to corrupt whoever wields it. It has been lost to time.",
+      @"ReplaceableTextures\CommandButtons\BTNmidnightGS.blp")
+    {
+      AddObjective(new ObjectiveControlLevel(UNIT_N00P_THE_ABYSS, 20));
```

Comment:
I think we also need to add a "hero is in rect" objective here and give the artifact to the hero when it spawns, since otherwise we're just creating a timer objective where the enemy is about as likely to be able to take the item as Black Empire is.

That's something I really should have done for the similar Ironforge quest too.

Reply:
Yep, check out Sentinel's Scepter of the Queen quest. It gives the Scepter to the unit in the region.

Reply:
Yup just like that!